### PR TITLE
TLP 8 Actual job vacancies

### PR DESCRIPTION
**Pull request by nepnep83**
Added actual job vacancies
Fixed some code
Testing
Moved code to different files

### CODE REVIEW

File: test_job_vacancies.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                          'activedate': {'start': '2023-05-15T15:31:00Z', 'end': '2023-06-14T15:31:00Z'},
+                                         'location': {'location': 'BR1 3NN', 'city': '', 'area': '', 'postcode': '',
+                                                      'country': ''},
+                                         'link': 'https://findajob.dwp.gov.uk/details/12181272'}]
+        mocked_info.return_value = [10, 'da1']
+        mocked_vacancies.return_value = [
+            {'id': 12181272, 'title': 'Plumber', 'summary': "Plumbers", 'company': 'Talent Finder',
+             'activedate': {'start': '2023-05-15T15:31:00Z', 'end': '2023-06-14T15:31:00Z'},
+                            'location': {'location': 'BR1 3NN', 'city': '', 'area': '', 'postcode': '', 'country': ''},
+                            'link': 'https://findajob.dwp.gov.uk/details/12181272'}]
```
Comment (Aidan23Smith) [inline test_job_vacancies.py:75]: There's a lot of repeated code here, could they be taken out as constants?

File: test_recommended_jobs.py [hunk 2]
```diff
@@ -147,21 +142,38 @@ def test_find_job(self, mocked_request):
...
+    @mock.patch('recommend_jobs.onet_interests')
+    @mock.patch('recommend_jobs.reverse_search')
+    @mock.patch('recommend_jobs.find_job')
+    def test_run(self, mocked_job, mocked_search, mocked_onet_interests,
+                 mocked_onet_skills, mocked_onet, mocked_soc, Mock):
+        expected_data = ['plumber']
+        mocked_job.return_value = ['plumber']
+        mocked_search.return_value = [1115]
+        mocked_onet_interests.return_value = [{'id': '1.B.1.a', 'name': 'Realistic', 'value': 1.33}]
+        mocked_onet_skills.return_value = [{'id': '2.A.1.a', 'name': 'Reading Comprehension', 'value': 3.0},
```
Comment (Aidan23Smith) [inline test_recommended_jobs.py:163]: Again, a lot of repetition.

Could you take a few skills/interests, or any of these jsons and put them as constant variables

File: test_job_vacancies.py (hunk 1 above)
Comment (nepnep83) [reply test_job_vacancies.py]: Done

File: test_recommended_jobs.py (hunk 2 above)
Comment (nepnep83) [reply test_recommended_jobs.py]: Done

**Review by Aidan23Smith — APPROVED** [review]

**Pull request merged**